USAREC_DS_Skills_Assess_20240903_V1.Rmd

```diff
@@ -0,0 +1,221 @@
+---
+title: "USAREC DS Skills Assessment submission Code"
+author: "Lydia Tanque"
+date: "2024-09-03"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+'rm(list = ls())' # Removed original code 
+
+# 1. Load the packages
+# require(tidyverse) # Removed function
+ 
+library(ggplot2)
+library(magrittr)
+library(dplyr)
+library(stringr)
+library(readr)
+library(caret)
+library(pheatmap)
+library(skimr)
+library(DBI) 
+library(RPostgres) 
+```
+
+```{r}
+# Create a DB connection ()
+con <- dbConnect(RPostgres::Postgres(), #Change this based on db type
+                  dbname = Sys.getenv("DB_NAME"),
+                  host = Sys.getenv("DB_HOST"),
+                  port = as.integer(Sys.getenv("DB_PORT")),
+                  user = Sys.getenv("DB_USER"),
+                  password = Sys.getenv("DB_PASSWORD"))
+```
+
+```{sql connection=con, echo=FALSE}
+-- Query database 
+SELECT 
+  unit_id, 
+  SUM(applicants) AS sum_of_applicants,
+  fiscal_year, 
+  fiscal_month
+  
+FROM 
```

Comment:
ok, I see where you did change the SQL in the `.Rmd` -- nice work and an approach I hadn't considered by converting it all to R. When we work with the pipelines here we do actually need to keep the scripts separate though.